parser.go

```diff
@@ -797,6 +797,19 @@ func (v *Value) GetStringBytes(keys ...string) []byte {
 	return s2b(v.s)
 }
 
+// GetString returns string value by the given keys path.
+//
+// Array indexes may be represented as decimal numbers in keys.
+//
+// An empty string is returned for non-existing keys path or for invalid value type.
+func (v *Value) GetString(keys ...string) string {
+	v = v.Get(keys...)
+	if v == nil || v.Type() != TypeString {
+		return ""
+	}
+	return v.s
```

Comment:
`v.s` cannot be returned as is, since it points to an internal byte slice - `Parser.b`, which is overwritten on each `Parser.Parse` call. The same applies to `GetStringBytes`, but strings are easier to mess up comparing to byte slices. So the  a copy of `v.s` must be returned to be safe:
```go
    return string(append([]byte(nil), v.s...))
```
But this leads to an allocation and a string copy for each `GetString` call. I'm unsure whether this function must be added.

Reply:
If you want obtaining strings without allocations, then just use something like the following:
```go
func getString(v *fastjson.Value) string {
    sb := v.GetStringBytes()
    return *(*string)(unsafe.Pointer(&sb))
}
```
But this construction must be used only if you are 100% sure what are you doing, otherwise you'll shoot in the foot. That's why I don't want adding zero-alloc `Value.GetString`

Reply:
I didn't know that v.s points to Parser.b. Even so this function could be added to avoid splicit string conversion or the usage of Value.Get(keys).String().

If you reject this pull request I will understand the reason.